src/service/performance-impl.js

```diff
@@ -75,7 +75,7 @@ export class Performance {
     this.win = win;
 
     /** @private @const {number} */
-    this.initTime_ = this.win.performance.now();
+    this.initTime_ = parseInt(this.win.performance.now(), 10);
```

Comment:
`parseInt` is for turning strings into ints, use `Math.round()`, `floor`, or `ceil`. But we don't need to do this here, just for the return value of `getDomHighResTimeStamp_`.